Zend/zend_API.h

```diff
@@ -1097,6 +1097,22 @@ ZEND_API zend_result zend_try_assign_typed_ref_res(zend_reference *ref, zend_res
 ZEND_API zend_result zend_try_assign_typed_ref_zval(zend_reference *ref, zval *zv);
 ZEND_API zend_result zend_try_assign_typed_ref_zval_ex(zend_reference *ref, zval *zv, bool strict);
 
+#define ZEND_SAFE_ASSIGN_VALUE(zv, value) do { \
```

Comment:
We already have very similar ``zend_assign_to_variable()`` in ``zend_execute.h``.
This one does almost the same except for reference support.
I think this sholud be named somehow more consistently and may be moved to zend_execute.h

The rest looks fine.